source/sessions/tests/unified/snapshot-sessions-unsupported-ops.json

```diff
@@ -457,7 +482,7 @@
             },
             {
               "commandFailedEvent": {
-                "commandName": "insert"
+                "listCollections": "insert"
```

Comment:
I think this should be:

     commandName: "listCollections"

Reply:
Fixed.